blocks/panel-color/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { PanelColor } from '@wordpress/components';
```

Comment:
It can be alsoimported as:
```js
import { PanelColor as PanelColorComponent } from '@wordpress/components';
```

Then we can name this component the same way as the file: `PanelColor`.